<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>657</width>
    <height>857</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">*{
	font-family: Arial;
	font-size: 20px;
}
#centralwidget{
	background-color: #0ca678;
}
QPushButton{
	height: 100px;
	border: 2px solid white;
	border-radius: 12px;
	color: white;
	background-color: transparent;
}
QPushButton:hover{
background-color: white; 
color: #0ca678;
}
</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>90</y>
      <width>411</width>
      <height>691</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QPushButton" name="btnBaseCalc">
       <property name="text">
        <string>Podstawowy kalkulator</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnAdvCalc">
       <property name="text">
        <string>Zaawansowane operacje</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnMatrixCalc">
       <property name="text">
        <string>Kalkulator macierzy</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnFunctions">
       <property name="text">
        <string>Przebieg zmienno≈õci funkcji</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
